docs/getting-started/cluster-setup.md

```diff
@@ -97,24 +99,25 @@ Other environment variables can also be customized for Screwdriver. For a full l
 
 
 ## Deploy Screwdriver
-You can check out the [Screwdriver API repo](https://github.com/screwdriver-cd/screwdriver) and look in the `kubernetes` directory for service and deployment definitions for running the Screwdriver API. Both files are required to deploy Screwdriver.
+You can check out the [Screwdriver Kubernetes repo](https://github.com/screwdriver-cd/kubernetes) and for service and deployment definitions for running the Screwdriver API. Both files are required to deploy Screwdriver.
```

Comment:
Ahh I see
Fixed! I just deleted that extra line. Service and Deployment used to be defined in separate files; in the new Kubernetes repo they are both defined in one file (api.yaml)